_aggregations/bucket/children.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Children
+parent: Bucket aggregations
+grand_parent: Aggregations
+nav_order: 15
+---
+
+# Children
+
+The `children` aggregation connects parent documents with their related child documents. This allows you to analyze relationships between different types of data in a single query, rather than having to run multiple queries and combine the results yourself.
+
+For example, if you have a parent-child relationship between authors, posts, and comments, you can analyze the relationships between the data different types (`authors`, `posts`, and `comments`) in a single query, without having to run multiple queries and combine the results manually. 
```

Comment:
@bowenlan-amzn Does this clarify the sample index and data comment?

Reply:
I think you need to provide the requests you used to index the data that correspond to below search request and response.